build.sbt

```diff
@@ -6,6 +6,13 @@ packageSummary := "Support Play APP"
 
 scalaVersion := "2.11.12"
 
+routesImport ++= Seq(
+  "utils.ThriftUtils.Implicits._",
```

Comment:
Hopefully some of these aren't needed if we can just blindly pass through the parameters to contributions-frontend